vocabularies/Org.OData.Capabilities.V1.xml

```diff
@@ -692,6 +711,36 @@ supported:
         </Property>
       </ComplexType>
 
+      <Term Name="ReadRestrictions" Type="Capabilities.ReadRestrictionsType" AppliesTo="EntitySet Singleton FunctionImport">
+        <Annotation Term="Core.Description" String="Restrictions on read operations: retrieve a collection, retrieve a single instance, invoke a function" />
+      </Term>
+      <ComplexType Name="ReadRestrictionsType">
+        <Property Name="Readable" Type="Edm.Boolean" Nullable="false" DefaultValue="true">
+          <Annotation Term="Core.Description" String="Entities can be retrieved" />
+          <Annotation Term="Core.LongDescription" String="This is only meaningful if the annotation is applied to an entity set or singleton." />
+        </Property>
+        <Property Name="CustomHeaders" Type="Collection(Capabilities.CustomParameter)">
+          <Annotation Term="Core.Description" String="Supported or required custom headers" />
+        </Property>
+        <Property Name="CustomQueryOptions" Type="Collection(Capabilities.CustomParameter)">
+          <Annotation Term="Core.Description" String="Supported or required custom query options" />
+        </Property>
+        <Property Name="Permissions" Type="Capabilities.Permission">
+          <Annotation Term="Core.Description" String="List of required scopes to invoke an action or function." />
+        </Property>
+      </ComplexType>
+      
+      <Term Name="OperationRestrictions" Type="Collection(Capabilities.OperationRestriction)" Nullable="false" AppliesTo="Action Function">
+        <Annotation Term="Core.Description" String="Restrictions for function or action operation" />
+      </Term>
+      <ComplexType Name="OperationRestriction">
+        <Property Name="CustomHeaders" Type="Collection(Capabilities.CustomParameter)">
+          <Annotation Term="Core.Description" String="Request header values allowed for function or action." />
+        </Property>
+        <Property Name="Permissions" Type="Capabilities.Permission">

```

Comment:
Move down to existing definition of ReadRestrictionsType in https://github.com/oasis-tcs/odata-vocabularies/pull/35/files#diff-40dc44aa44d41fa1bf22798bb68972a4L730, ideally after"Readable" and before "CustomHeaders", the same position as in other types.